src/simularium/VisGeometry/rendering/InstancedMesh.ts

```diff
@@ -133,6 +156,7 @@ class InstancedMesh {
     beginUpdate(): void {
         this.isUpdating = true;
```

Comment:
it's true, no one is using it now. I put it there as a sort of guard in case someone tries to make the wrong function calls in between calling beginUpdate and endUpdate (or outside of that).  I suppose I should either enforce it in code or remove it huh :)